assets/src/editor/events/ticket-assignments-manager/filters/ticket-filters.js

```diff
@@ -14,22 +22,25 @@ const { FormColumn } = twoColumnAdminFormLayout;
  * @function
  * @param {boolean} showArchivedTickets
  * @param {boolean} showExpiredTickets
- * @param {Function} toggleArchivedTickets
- * @param {Function} toggleExpiredTickets
- * @param {boolean} showDateFilters
- * @param {boolean} showTicketFilters
+ * @param {number} ticketFiltersOffset
+ * @param {Function} setFilter
  * @return {Object} rendered ticket filter toggles
  */
 const TicketFilters = ( {
 	showArchivedTickets,
 	showExpiredTickets,
-	toggleArchivedTickets,
-	toggleExpiredTickets,
-	showDateFilters,
-	showTicketFilters,
+	ticketFiltersOffset,
+	setFilter,
 } ) => {
-	const ticketFiltersOffset = showDateFilters && showTicketFilters ? 0 : 7;
-	return showTicketFilters ? (
+	const toggleArchivedTickets = useCallback(
+		() => setFilter( showArchivedTicketsAction ),
+		[]
+	);
```

Comment:
This should have `setFilter` and `showArchivedTicketsActions` as dependencies.

Reply:
see response above